sdk/ml/azure-ai-ml/azure/ai/ml/entities/_monitoring/signals.py

```diff
@@ -260,7 +260,7 @@ class ReferenceData(RestTranslatableMixin):
     def __init__(
         self,
         *,
-        input_data: Input,
+        input_data: Optional[Input] = None,
```

Comment:
Can you update the docstring type as well?

Reply:
Sorry i was using this PR to generate a private whl for the sdk

Reply:
No worries

Reply:
No worries